### PR TITLE
[ios][ExposureNotification] Update bindings to Xcode 11.6 Beta 1

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

File: iOS/ExposureNotification/source/ExposureNotification.csproj [hunk 1]
```diff
@@ -5,8 +5,8 @@
     <IsBindingProject>true</IsBindingProject>
     <RootNamespace>ExposureNotification</RootNamespace>
     <AssemblyName>Xamarin.iOS.ExposureNotification</AssemblyName>
-    <AssemblyVersion>1.0.0.0</AssemblyVersion>
-    <FileVersion>1.0.0.0</FileVersion>
+    <AssemblyVersion>1.1.0.0</AssemblyVersion>
+    <FileVersion>1.1.0.0</FileVersion>
```
Comment (mattleibow) [inline iOS/ExposureNotification/source/ExposureNotification.csproj]: Do we want to do this? Do we not keep the actual assembly version at 1.0.0.0 and just bump the information part?

File: iOS/ExposureNotification/source/ApiDefinition.cs [hunk 2]
```diff
@@ -106,6 +114,15 @@ interface ENExposureConfiguration {
 		[Export ("minimumRiskScore")]
 		byte MinimumRiskScore { get; set; }
 
+		[Introduced (PlatformName.iOS, 13, 6)]
+		[Export ("minimumRiskScoreFullRange")]
+		double MinimumRiskScoreFullRange { get; set; }
+
+		[Introduced (PlatformName.iOS, 13, 6)]
+		[BindAs (typeof (int []))]
```
Comment (mattleibow) [inline iOS/ExposureNotification/source/ApiDefinition.cs:122]: What does the `[BindAs (typeof (int []))]` actually do when the return type is `NSNumber[]`?

File: iOS/ExposureNotification/source/ExposureNotification.csproj (hunk 1 above)
Comment (dalexsoto) [inline iOS/ExposureNotification/source/ExposureNotification.csproj]:
```suggestion
    <AssemblyVersion>1.0.0.0</AssemblyVersion>
    <FileVersion>1.0.0.0</FileVersion>
```

File: iOS/ExposureNotification/source/ApiDefinition.cs (hunk 2 above)
Comment (dalexsoto) [reply iOS/ExposureNotification/source/ApiDefinition.cs:122]: This means that we will take care of the conversion of `NSNumber []` <-> `int []` automatically in the binding code so the signature will become

```csharp
int [] AttenuationDurationThresholds { get; }
```

So you do not need to worry about the NSNumber [] detail

Comment (mattleibow) [reply iOS/ExposureNotification/source/ApiDefinition.cs:122]: That is very nice.

**Review by mattleibow — APPROVED** [review]

**Pull request merged**